docs/src/components/hardware-list.tsx

```diff
@@ -0,0 +1,149 @@
+import React from "react";
+
+import {
+  Board,
+  HardwareMetadata,
+  Interconnect,
+  Shield,
+} from "../hardware-metadata";
+
+interface HardwareListProps {
+  items: HardwareMetadata[];
+}
+
+type ElementOrString = JSX.Element | string;
+
+function itemIds(item: HardwareMetadata) {
+  if (item.type == "board" || item.type == "shield") {
+    let nodes = (item.siblings || [item.id])
+      .map<ElementOrString>((id) => <code key={id}>{id}</code>)
+      .reduce(
+        (prev, curr, index) => [...prev, index > 0 ? " and " : "", curr],
```

Comment:
Future-proof for more than 2 siblings. Feel free to ignore.
```suggestion
        (prev, curr, index) => [...prev, index > 0 ? ", " : "", curr],
```

Reply:
Fixed.